cpp/example_code/transcribe/get_transcript.cpp

```diff
@@ -0,0 +1,150 @@
+#include <aws/core/platform/FileSystem.h>
+#include <fstream>
+#include <chrono>
+#include <thread>
+#include <array>
+
+using namespace Aws;
+using namespace Aws::TranscribeStreamingService;
+using namespace Aws::TranscribeStreamingService::Model;
+
+//TODO(User): Update path to location of local .wav test file, if necessary.
+static const Aws::String FILE_NAME{MEDIA_DIR "/transcribe-test-file.wav"};
+static const int BUFFER_SIZE = 1024;
+static const int END_OF_STREAM_SLEEP_SECONDS = 10;
+
+//snippet-start:[transcribe.cpp.stream_transcription_async.code]
+int main() {
+    Aws::SDKOptions options;
+    options.loggingOptions.logLevel = Aws::Utils::Logging::LogLevel::Trace;
+    Aws::InitAPI(options);
+    {
+        //TODO(User): Set to the region of your AWS account.
+        const Aws::String region = Aws::Region::US_WEST_2;
+
+        Aws::Utils::Threading::Semaphore received(0 /*initialCount*/, 1 /*maxCount*/);
+
+        //Load a profile that has been granted AmazonTranscribeFullAccess AWS managed permission policy.
+        Aws::Client::ClientConfiguration config;
+#ifdef _WIN32
+        // ATTENTION: On Windows with AWS SDK version 1.9, this example only runs if the SDK is built
+        // with the curl library.  (9/15/2022)
+        // See the accompanying ReadMe.
+        // See "Building the SDK for Windows with curl".
+        // https://docs.aws.amazon.com/sdk-for-cpp/v1/developer-guide/setup-windows.html
+        //TODO(User): Update to location of your .crt file.
+        config.caFile = "C:/curl/bin/curl-ca-bundle.crt";
+#endif
+        config.region = region;
+
+        TranscribeStreamingServiceClient client(config);
+        StartStreamTranscriptionHandler handler;
+        handler.SetOnErrorCallback(
+                [](const Aws::Client::AWSError<TranscribeStreamingServiceErrors> &error) {
+                        std::cerr << "ERROR: " + error.GetMessage() << std::endl;
+                });
+        //SetTranscriptEventCallback called for every 'chunk' of file transcripted.
+        // Partial results are returned in real time.
+        handler.SetTranscriptEventCallback([&received](const TranscriptEvent &ev) {
+                for (auto &&r: ev.GetTranscript().GetResults()) {
+                    if (r.GetIsPartial()) {
+                        std::cout << "[partial] ";
+                    }
+                    else {
+                        std::cout << "[Final] ";
+                    }
+                    for (auto &&alt: r.GetAlternatives()) {
+                        std::cout << alt.GetTranscript() << std::endl;
+                    }
+                }
+                received.Release();
+        });
+
+        StartStreamTranscriptionRequest request;
+        request.SetMediaSampleRateHertz(8000);
+        request.SetLanguageCode(LanguageCode::en_US);
+        request.SetMediaEncoding(
+                MediaEncoding::pcm); //wav and aiff files are PCM formats
```

Comment:
add ending punctuation to inline code comment

Reply:
added